qt/gen_qstringlist.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
```

Comment:
The rest of the code sees QStringList as being a `KnownTypedefs` alias for `QList<QString>` to get a nice native type projection instead.

I guess it's showing up because of the `file` changes? Might be better to block in Widgets_AllowHeader(),

Reply:
[f85157b](https://github.com/mappu/miqt/pull/226/commits/f85157b52d813b02301b703d85119345547f8d36)